server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -169,7 +169,8 @@ public static void validateIndexOrAliasName(String index, BiFunction<String, Str
         }
         if (index.contains(":")) {
             deprecationLogger.deprecated("index or alias name [" + index +
-                            "] containing ':' is deprecated and will not be supported in Elasticsearch 7.0+");
+                            "] containing ':' is deprecated. Elasticsearch 7.0+ will read, " +
```

Comment:
nit s/7.0+/7.x

this can perhaps imply this is strictly limited to 7.x without having to say explicitly 8.x drops reads